atomicfu/src/jsMain/kotlin/kotlinx/atomicfu/AtomicFU.kt

```diff
@@ -7,20 +7,28 @@
 package kotlinx.atomicfu
 
 import kotlin.reflect.KProperty
-import kotlin.reflect.KProperty0
+import kotlinx.atomicfu.TraceBase.None
 
 @JsName("atomic\$ref\$")
 public actual fun <T> atomic(initial: T, trace: TraceBase): AtomicRef<T> = AtomicRef<T>(initial)
 
+public actual inline fun <T> atomic(initial: T): AtomicRef<T> = atomic(initial, None)
```

Comment:
Do we need `JsName` on these overloads as well?

Reply:
No, it is inlined and the overload with the mangled name is called